jax/_src/numpy/array_creation.py

```diff
@@ -59,7 +59,12 @@ def zeros(shape: Any, dtype: DTypeLike | None = None, *,
     dtype: optional dtype for the created array; defaults to float32 or float64
       depending on the X64 configuration (see :ref:`default-dtypes`).
     device: (optional) :class:`~jax.Device` or :class:`~jax.sharding.Sharding`
-      to which the created array will be committed.
+      to which the created array will be committed. This argument exists for
+      compatibility with the :ref:`python-array-api`.
+    out_sharding: (optional) :class:`~jax.sharding.PartitionSpec` or
```

Comment:
LGTM, but can you also link to the explicit_sharding guide here? `out_sharding` is meant for sharding-in-types mode.